arenadata/Dockerfile

```diff
@@ -18,23 +18,12 @@ RUN dnf makecache && \
     dnf -y install libicu perl-ExtUtils-Embed perl-Env perl-JSON && \
     dnf -y install perl-IPC-Run perl-Test-Base libxslt-devel openldap-devel && \
     dnf -y install python39-psycopg2 python39-psutil python39-pyyaml python39-lxml && \
+    dnf -y install llvm-devel clang && \
```

Comment:
Is it necessary to run dnf one more time? Can we add these two packages to the existing package list, for example in the 14th line? I think the fewer times we run dnf, the faster the installation is.

Reply:
I think it would be justified if we had one dnf run for all the packages but it's called multiple times so I think it's better to leave clang and llvm on a separate line to show that they're connected and keep diff obvious so one would see what changes were made at a glance. It wouldn't be so clear if you left it on the same line with python